src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -156,6 +156,8 @@ class ListOffersQueryModel(BaseModel):
     venue_id: Optional[int]
     type_id: Optional[str]
     creation_mode: Optional[str]
+    period_beginning_date: Optional[str]
+    period_ending_date: Optional[str]
```

Comment:
```suggestion
    period_ending_date: Optional[datetime]
```
Jsuis pas sûr de mon coup mais ça renforcerait la signature.